Makefile

```
@@ -0,0 +1,36 @@
+.PHONY: all
+all: help
+
+.PHONY: clean
+clean:
+	rm -f .coverage coverage.xml writer.pickle
+	rm -rf .pytest_cache build dist htmlcov test-reports
```

Maybe we can also add `docs/_build`:

```suggestion
	rm -rf .pytest_cache build dist htmlcov test-reports docs/_build
```